lib/control/v1/unseal.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const bodyParser = require('body-parser');
+const reject = require('../validation/methodNotAllowed');
+const HTTP_OK = 200;
+
+function Unseal(app) {
+  app.use(bodyParser.json());
+
+  app.post('/v1/unseal',
+    (req, res, next) => {
+      if (req.body){
+        Config.set('vault:token', req.body);
```

Comment:
It was pointed out to me that I don't need the flag as I can I check to see if the token exists instead of using a boolean. So I removed all the unseal flags and now check if the token exists.